.github/workflows/nightly.yml

```diff
@@ -2,7 +2,7 @@ name: Nightly checks
 
 on:
   schedule:
-    - cron: "0 0 * * *" # every day at midnight
+    - cron: "0 22 * * *" # daily at 10:00 PM
```

Comment:
Out of curiosity, why?

Reply:
I've increased the timeout to 10 hours, it shouldn't take anywhere near that, but if it did, it's probably better if it finishes before most of us get back in the office in the morning